temi_code/curate_my_heart_counts_public_BRIDGE.R

```diff
@@ -343,7 +435,7 @@ main <- function() {
   df <- readRDS("df.rds")
   my_heart_counts <- df %>%
     mutate_participant_week_day() %>%
-    mutate_local_time() %>% # adds state from demographics file
+    mutate_local_time() %>% # adds zipcodes from satisfied survey
```

Comment:
I think this adds the zip, lat, lon to the df based on https://github.com/Sage-Bionetworks/digitalHealth_RetentionAnalysis_PublicRelease/blob/master/curation_scripts/curate_my_heart_counts_public.R#L131, but would be curious what the columns are in the `my_heart_counts ` dataframe in RStudio's explorer

Reply:
yeah you're right it should say adds zip, lat, lon, timezone, and state too